app/tag-management/annotation/pipes/filter-tags.pipe.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable, Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'hipTagsFilter'
+})
+
+@Injectable()
+export class TagsFilter implements PipeTransform {
```

Comment:
File name and class name should be consistent. please rename the file to tags-filter.pipe.ts